Resources/doc/.doctor-rst.yaml

```diff
@@ -0,0 +1,47 @@
+rules:
+    no_inheritdoc: ~
+    avoid_repetetive_words: ~
+    blank_line_after_directive: ~
+    short_array_syntax: ~
+    typo: ~
+    replacement: ~
+    versionadded_directive_should_have_version: ~
+    deprecated_directive_should_have_version: ~
+    max_blank_lines:
+        max: 2
+    american_english: ~
+    valid_use_statements: ~
+    no_composer_req: ~
+    composer_dev_option_not_at_the_end: ~
+    no_php_open_tag_in_code_block_php_directive: ~
+    no_blank_line_after_filepath_in_php_code_block: ~
+    no_blank_line_after_filepath_in_yaml_code_block: ~
+    no_blank_line_after_filepath_in_xml_code_block: ~
+    no_blank_line_after_filepath_in_twig_code_block: ~
+    php_prefix_before_bin_console: ~
+    use_deprecated_directive_instead_of_versionadded: ~
+    space_before_self_xml_closing_tag: ~
+    ensure_order_of_code_blocks_in_configuration_block: ~
+    lowercase_as_in_use_statements: ~
+    ordered_use_statements: ~
+    no_namespace_after_use_statements: ~
+    replace_code_block_types: ~
+    use_https_xsd_urls: ~
+    blank_line_before_directive: ~
+    extension_xlf_instead_of_xliff: ~
+    valid_inline_highlighted_namespaces: ~
+    indention: ~
+    unused_links: ~
+    no_app_bundle: ~
+
+    versionadded_directive_major_version:
+        major_version: 3
+
+    versionadded_directive_min_version:
+        min_version: '3.0'
+
+    deprecated_directive_major_version:
+        major_version: 3
+
+    deprecated_directive_min_version:
+        min_version: '3.0'
```

Comment:
After up merging into 4.x this must be change accordingly. Same for `master`, if this will be next 5.0, otherwise not.